publisher/product/glossary/build.sh

```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
 #
 function publishProductGlossaryContent() {
 
-  logRule "Publishing the content files of the glossary product"
-
-  require ontology_product_tag_root || return $?
-  require glossary_product_tag_root || return $?
-
-  local numberOfProductionLevelOntologyFiles=0
+	logRule "Publishing the content files of the glossary product"
```

Comment:
the formatting is not consistent with that used elsewhere in the code - compare with [deleted line 118](https://github.com/edmcouncil/ontology-publisher/pull/24/files#diff-4be3c5b9229ba89792c3fa046edbbc35d1c6d8ee76d95206275588eec514f315L118)

Reply:
This should be fixed by the patch.